apps/admin_app/assets/js/views/product/edit.js

```diff
@@ -31,6 +32,25 @@ export function imageOnEnter(){
   });
 }
 
+export function defaultImage(){
```

Comment:
@jyotigautam shouldn't this method have a more meaningful name, based on what it does. I see that this method sets an image as default image. Based on this I would name the method as `setDefaultImage` or `updateDefaultImage` let me know what you think?

Reply:
Yes, defaultImage() doesn't make it clear what exactly is done with the image. You are right, fixing this in the next commit.